test/integration/full/run-partial/after-method.js

```diff
@@ -23,6 +23,11 @@ describe('run-partial, after-method', function() {
         var axeRunResult = results[1];
         assert.lengthOf(axeRunPartialResult.violations, 0);
 
+        // it appears the selenium webdriver browser message "Chrome is being controlled by
+        // automated software" changes the window height when it appears
+        // between running the two axe runs
+        axeRunPartialResult.testEnvironment = axeRunResult.testEnvironment;
+
```

Comment:
I don't think these comments are necessary. Put 'm on the PR if you like, so we can find it. It is not relevant for understanding this code.